app/assets/stylesheets/application.css

```diff
@@ -10,6 +10,10 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
```

Comment:
Unless I misunderstood, Max said he'd customised the css so I should copy it over.